flow/visualize/time_space_diagram.py

```diff
@@ -516,45 +516,16 @@ def _get_abs_pos_1_edge(edges, rel_pos, edge_starts):
     return ret
 
 
-if __name__ == '__main__':
-    # create the parser
-    parser = argparse.ArgumentParser(
-        formatter_class=argparse.RawDescriptionHelpFormatter,
-        description='[Flow] Generates time space diagrams for flow networks.',
-        epilog='python time_space_diagram.py </path/to/emission>.csv '
-               '</path/to/flow_params>.json')
-
-    # required arguments
-    parser.add_argument('emission_path', type=str,
-                        help='path to the csv file.')
-    parser.add_argument('flow_params', type=str,
-                        help='path to the flow_params json file.')
-
-    # optional arguments
-    parser.add_argument('--steps', type=int, default=1,
-                        help='rate at which steps are plotted.')
-    parser.add_argument('--title', type=str, default='Time Space Diagram',
-                        help='rate at which steps are plotted.')
-    parser.add_argument('--max_speed', type=int, default=8,
-                        help='The maximum speed in the color range.')
-    parser.add_argument('--min_speed', type=int, default=0,
-                        help='The minimum speed in the color range.')
-    parser.add_argument('--start', type=float, default=0,
-                        help='initial time (in sec) in the plot.')
-    parser.add_argument('--stop', type=float, default=float('inf'),
-                        help='final time (in sec) in the plot.')
-
-    args = parser.parse_args()
-
+def make_ts_diagram(flow_params, emission_path, min_speed, max_speed, start, stop, title):
```

Comment:
all i did here was move the code in main method into a separate method so that i could call it from another file